ingest-app/src/cmr/ingest/api/routes.clj

```diff
@@ -252,13 +252,6 @@
 
 ;; TODO: Generic work - should this change to be more dynamic, gen type in front?
```

Comment:
comment should go right?

Reply:
probably, but this wasn't my comment and so i am not 100% sure i am in a position to make that call. also, i thought that wasn't in scope for this ticket --  i assumed that we are going to be making another ticket/PR for the multitude of "clean up tasks" such as this. i am happy to remove this if someone confirms it should be removed.

Reply:
went ahead and removed this